packages/admin/admin/src/table/TableBodyRow.tsx

```diff
@@ -1,40 +1,56 @@
-import { ComponentsOverrides, TableRow, Theme } from "@mui/material";
+import { ComponentsOverrides, TableRow } from "@mui/material";
+import { css, styled, Theme, useThemeProps } from "@mui/material/styles";
 import { TableRowProps } from "@mui/material/TableRow";
-import { createStyles, WithStyles, withStyles } from "@mui/styles";
-import * as React from "react";
+import { ThemedComponentBaseProps } from "helpers/ThemedComponentBaseProps";
+import React from "react";
 
 /**
  * @deprecated Use MUI X Data Grid in combination with `useDataGridRemote` instead.
  */
 export type TableBodyRowClassKey = "root" | "even" | "odd";
 
+type OwnerState = Pick<TableBodyRowProps, "index" | "hideTableHead">;
```

Comment:
The only thing really needed in the `ownerState` would be `isOdd`. 
I would change this and move the logic from the `overridesResolver` into the component.